ui/raidboss/popup-text.ts

```diff
@@ -1266,20 +1266,21 @@ export class PopupText {
   _onTriggerInternalTTS(triggerHelper: TriggerHelper): void {
     if (!triggerHelper.groupSpokenAlertsEnabled || typeof triggerHelper.ttsText === 'undefined') {
       let result = undefined;
-      if (triggerHelper.triggerOptions.TTSText) {
+      if (triggerHelper.triggerOptions.TTSText !== undefined) {
         result = triggerHelper.valueOrFunction(triggerHelper.triggerOptions.TTSText);
       } else if (triggerHelper.trigger.tts !== undefined) {
         // Allow null/false/NaN/0/'' in this branch.
         result = triggerHelper.valueOrFunction(triggerHelper.trigger.tts);
       } else if (triggerHelper.response) {
         const resp: ResponseField<RaidbossData, Matches> = triggerHelper.response;
-        if (resp.tts)
+        if (resp.tts !== undefined)
           result = triggerHelper.valueOrFunction(resp.tts);
       }
 
       // Allow falsey values to disable tts entirely
       // Undefined will fall back to defaultTTSText
       if (result !== undefined) {
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```

Comment:
This is a case where we actually want to allow the falsey behavior, as indicated by the comment above, so I elected to disable the rule for this line instead.